lib/kino_db/sql_cell.ex

```diff
@@ -222,6 +256,22 @@ defmodule KinoDB.SQLCell do
     end
   end
 
+  defp to_explorer_quoted(attrs, next) do
+    {query, params} = parameterize(attrs["query"], next)
+    opts_args = query_opts_args(attrs)
+    data_frame_alias = attrs["data_frame_alias"]
+
+    quote do
+      unquote(quoted_var(attrs["result_variable"])) =
+        unquote(data_frame_alias).from_query!(
+          unquote(quoted_var(attrs["connection"]["variable"])),
+          unquote(quoted_query(query)),
+          unquote(params),
+          unquote_splicing(opts_args)
```

Comment:
Maybe we should remove options as we don't use them at all?

```suggestion
    data_frame_alias = attrs["data_frame_alias"]

    quote do
      unquote(quoted_var(attrs["result_variable"])) =
        unquote(data_frame_alias).from_query!(
          unquote(quoted_var(attrs["connection"]["variable"])),
          unquote(quoted_query(query)),
          unquote(params)
```

Your call!